### ISSUE TITLE
Can't enable when no internet

### ISSUE THREAD

**Issue by rlaurente** [Uepaa-AG/p2pkit-cordova#2]
Is this possible to work even if wifi and mobile data is disabled. It seems it rely heavily in your server ?

**Comment by uepaa** [comment]
Yes it does - even discovery is physically p2p you need to be connected to
the internet with p2pkit for handling the discovery engine. Both for speed
and scalability.

Thanks M

2017-12-13 8:20 GMT+01:00 rlaurente <notifications@github.com>:

**Comment by rlaurente** [comment]
![screen shot 2017-12-13 at 4 41 45 pm](https://user-images.githubusercontent.com/1804380/33929461-916121da-e024-11e7-8f3d-4e3d67f2100a.png)
I see it didn't enabled, you mean it can still detect nearby devices even if it fails to enabled ?
